systemvm/patches/debian/config/opt/cloud/bin/vr_cfg.sh

```diff
@@ -27,39 +27,29 @@ log_it() {
     echo "$(date) : $*" >> $log
 }
 
-while getopts 'c:' OPTION
-do
-  case $OPTION in
-      c) cfg="$OPTARG"
-          ;;
-  esac
-done
+while getopts 'c:' OPTION; do
+    case $OPTION in
+        c) cfg="$OPTARG" ;;
+esac; done
```

Comment:
especially this line is concluding two levels of indentation at once. Please keep them for readability